cogs/TextFilter.py

```diff
@@ -0,0 +1,701 @@
+        elif len(ctx.message.channel_mentions) > 0:
+            ignore_id = ctx.message.channel_mentions[0].id
+        else:
+            raise Exception("No ignore mention found")
+        self.tf_database_manager.remove_ignore(ctx.guild.id, ignore_id)
+        await ctx.channel.send("Ignore removed: " + str(ignore))
+        return
+    
+    @commands.command(name="ignoreList", aliases=["list_ignored", "listIgnored"])
+    @commands.check(KoalaBot.is_admin)
+    @commands.check(text_filter_is_enabled)
+    async def list_ignored(self, ctx):
+        """
+        Get a list all ignored users/channels
+        :param ctx: The discord context
+        :return:
+        """
+        ignored= self.tf_database_manager.get_all_ignored(ctx.guild.id)
+        await ctx.channel.send(embed=build_ignore_list_embed(self, ctx, ignored))
+
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        """
+        Upon receiving a message, it is checked for filtered text and is deleted.
+        :param message: The newly received message
+        :return:
+        """
+        if message.content.startswith(KoalaBot.COMMAND_PREFIX+"filter") or \
+                message.content.startswith(KoalaBot.COMMAND_PREFIX+"unfilter"):
+            return
+        elif str(message.channel.type) == 'text' and message.channel.guild is not None:
+            censor_list = self.tf_database_manager.get_filtered_text_for_guild(message.channel.guild.id)
+            for word, filter_type, is_regex in censor_list:
+                if (word in message.content or re.search(word, message.content)) and not is_ignored(self, message):
+                    if filter_type == "risky":
+                        await message.author.send("Watch your language! Your message: '*"+message.content+"*' in "+
+                                                  message.channel.mention+" contains a 'risky' word. "
+                                                  "This is a warning.")
+                        return
+                    elif filter_type == "banned":
+                        await message.author.send("Watch your language! Your message: '*"+message.content+"*' in "+
+                                                  message.channel.mention+" has been deleted by KoalaBot.")
+                        await send_to_moderation_channels(self, message)
+                        await message.delete()
+                        return
+
+
+def is_ignored(self, message):
+    """
+    Checks if the user/channel should be ignored
+    :param message: The newly received message
+    :return boolean if should be ignored or not:
+    """
+    ignore_list_users = self.tf_database_manager.get_ignore_list_users(message.guild.id)
+    ignore_list_channels = self.tf_database_manager.get_ignore_list_channels(message.guild.id)
+    return message.channel.id in ignore_list_channels or message.author.id in ignore_list_users
+
+
+def setup(bot: KoalaBot) -> None:
```

Comment:
small change, if possible it'd be nice to have this function at the bottom of the file so it lines up with all the other cogs, allows anyone to find this method for edits really easy.